app/src/main/java/chat/rocket/android/renderer/MessageRenderer.java

```diff
@@ -45,7 +45,7 @@ public MessageRenderer avatarInto(ImageView imageView, String hostname) {
    * show Username in textView.
    */
   public MessageRenderer usernameInto(TextView textView) {
-    userRenderer.usernameInto(textView);
+    userRenderer.usernameInto(textView, object.getAlias());
```

Comment:
like avatar, it would be better to set alias text here, independent of UserRenderer.

```
if (!TextUtils.isEmpty(object.getAlias)) {
  setAliasTextInto(object.getAlias(), object.getUser.getName(), textView);
} else {
  userRenderer.usernameInto(textView);
}
```